src/main/java/org/cascadebot/cascadebot/data/objects/GuildSettingsMusic.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.data.objects;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.cascadebot.cascadebot.commandmeta.Module;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@SettingsContainer(module = Module.MUSIC)
+@Getter
+@Setter
+@NoArgsConstructor(access = AccessLevel.PUBLIC)
+public class GuildSettingsMusic {
+
+    @Setting
+    private boolean preserveVolume = false;
+
+    @Setting
+    private boolean preserveEqualizer = false;
```

Comment:
I believe both of these options should be set to true by default imo

Reply:
I thought because both of these are donation stuffs they should be off by default. Granted I should really figure out how this needs to work. We don't want people who stop donating to keep being able to have a volume level.

Reply:
Hmm fair point, we'll discuss this :)